master/pkg/searcher/search_method.go

```diff
@@ -69,38 +69,31 @@ func NewSearchMethod(c expconf.SearcherConfig) SearchMethod {
 		return newRandomSearch(*c.RawRandomConfig)
 	case c.RawGridConfig != nil:
 		return newGridSearch(*c.RawGridConfig)
-	case c.RawAsyncHalvingConfig != nil:
-		if c.RawAsyncHalvingConfig.StopOnce() {
-			return newAsyncHalvingStoppingSearch(*c.RawAsyncHalvingConfig, c.SmallerIsBetter())
-		}
-		return newAsyncHalvingSearch(*c.RawAsyncHalvingConfig, c.SmallerIsBetter())
```

Comment:
Why did you delete the AsyncHalvingConfig case?

Reply:
i thought `async_halving` was a deprecated searcher and has been for a long time, so i removed it. but now i'm not so sure. let me circle back on this.

Reply:
I thought we talked about this?  I thought async_halving is just like adaptive except for it has a manually-chosen time constant.  Or rather, adaptive is like multiple trees of async_halving with different time constants, if you don't happen to know which time constant is right for your model.

Reply:
yeah, i remember now, you're right. i added it back. and since we're supporting it, i'll make a note to add it to our docs, since we don't officially advertise that we support it.